developer/12.0/guides/lexical-models/tutorial/step-2.php

```diff
@@ -2,93 +2,34 @@
   require_once('includes/template.php');
 
   head([
-    'title' => "Step 2: Get some language data",
-    'css' => ['template.css', 'index.css', 'prism.css', 'dev-lexical-models.css']
+    'title' => "Step 2: Creating a lexical model project"
   ]);
 ?>
 
-<h1>Step 2: Get some language data</h1>
+<h1>Step 2: Creating a lexical model project</h1>
 
-<p>To predict words in your language, a lexical model needs to know the words in your language!</p>
+<h2>Create the new project</h2>
+<p>Start Keyman Developer. From the Project menu, select "New Project" and then "Wordlist Lexical Model"</p>
 
+<p class='context-main-image'><img src="<?= cdn('img/developer/120/ui/frmNewLMProject.png') ?>" alt="New LM Project dialog" /></p>
 
-<p> Keyman Developer understands how to read words in a <strong>TSV file</strong>.
-This kind of file can be saved from a <strong>spreadsheet</strong> application like <a
-href="https://sheets.google.com/">Google Sheets</a> or
-<a href="https://products.office.com/en/excel">Microsoft Excel</a>.
-Other users may also use <strong>language data management software</strong>
-like <a href="https://software.sil.org/fieldworks/">SIL FieldWorks Language
-Explorer (FLEx)</a> or <a href="https://software.sil.org/wesay/">SIL WeSay</a>
-to export an appropriate TSV file.</p>
+<h2>New Project Parameters</h2>
+<p> To create a lexical model project, we will need the following identifiers: </p>
 
-<aside>
-  <p><strong>For advanced users</strong>: Ultimately, what Keyman Developer
-  requires is a tab-separated values (TSV) file in a specfic format described in
-  the <a href="../../../reference/file-types/tsv.php">file types reference</a>.
-  Refer to this reference file if you are coding your own exporter.
-  </p>
-</aside>
+<ul>
+  <li>Author Name - a short unique identifier, such as <code>nrc</code> or <code>sil</code></li>
+  <li>Model Name - an additional component to distinguish multiple models for a given language. For example <code>sencoten</code></li>
+  <li>Languages - the canonical BCP 47 tag for the language. For example <code>str</code> for the Straits Salish (SENĆOŦEN) language</li>
+</ul>
 
-<h2>Example</h2>
+<p class="hint">
+  <b>Note:</b>These fields are limited to the following characters: lower-case letters a-z, numerals 0-9,
```

Comment:
```suggestion
  <b>Note:</b> These fields are limited to the following characters: lower-case letters a-z, numerals 0-9,
```

Also, when using Keyman Developer, most of the options above will automatically ensure that model IDs are made up of the correct characters.

Reply:
Yeah, I'll remove the notice